models/observation.go

```diff
@@ -0,0 +1,82 @@
+package models
+
+// ObservationDoc represents information (metadata) relevant to a version
+type ObservationDoc struct {
+	Dimensions          map[string]Options `json:"dimensions,omitempty"`
+	Context             string             `json:"context,omitempty"`
+	Links               *ObservationLinks  `json:"links,omitempty"`
+	Observation         string             `json:"observation,omitempty"`
+	ObservationMetadata map[string]string  `json:"observation_level_metadata,omitempty"`
+	UsageNotes          *[]UsageNote       `json:"usage_notes,omitempty"`
+}
+
+// ObservationLinks represents a link object to list of links relevant to the observation
+type ObservationLinks struct {
+	DatasetMetadata *LinkObject `json:"dataset_metadata,omitempty"`
+	Self            *LinkObject `json:"self,omitempty"`
+	Version         *LinkObject `json:"version,omitempty"`
+}
+
+// Options is a an object containing a list of link onjects that refer to the
+// code url for that dimension option
+type Options struct {
+	LinkObjects []*LinkObject `json:"options,omitempty"`
```

Comment:
Given that each dimension can only have one option, this doesn't need to be an array? even when we handle wildcarding, it'd just be the list endpoint, and still a single link.

Reply:
You are right, and I was only thinking of the future and playing it safe; if ONS decide to filter on multiple options for a single dimension, this would break backwards compatibility unless they implemented versioning of resources.

I'll wait for your response before changing it or not